mwclient/client.py

```diff
@@ -1507,16 +1819,16 @@ def ask(self, query, title=None):
             >>>         print(title)
             >>>         print(data)
         """
-        kwargs = {}
+        kwargs = {}  # type: Dict[str, Any]
```

Comment:
since the only thing we ever put in `kwargs` is the title, this could just be `Dict[str, str]`, right?

Actually, er, isn't that code wrong and the check should be `if title is not None:`?

Reply:
Yeah, that makes sense. However, I would like to avoid mixing typing with functional changes. Let's create a separate PR either changing this condition or removing the parameter altogether. The API:Ask doesn't even have a `title` parameter from what I can see, neither in the initial nor the current implementation 😅:

- Initial implementation: https://github.com/SemanticMediaWiki/SemanticMediaWiki/pull/3052/files#diff-40638df1e5264aa7030d823f1f90c37081adfd76dd8da007f76d7bbc194987b1L50-L57
- Current implementation: https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/d3ef92592b4366d7a4c8cfc1c858956f4f5bfc60/src/MediaWiki/Api/Ask.php#L55-L67
- Docs: https://www.semantic-mediawiki.org/wiki/Help:API:ask

Reply:
OK. I don't mind which way around you want to do it (fix/drop it before this, or after).